content/docs/csidriver/release/powerflex.md

```diff
@@ -18,6 +19,7 @@ description: Release notes for PowerFlex CSI driver
 ### Fixed Issues
 
 - [#1014 - [BUG]: Missing error check for os.Stat call during volume publish](https://github.com/dell/csm/issues/1014)
+- [#1030 - [BUG]:Comment out duplicate entries in the sample secret.yaml file](https://github.com/dell/csm/issues/1030)
```

Comment:
lack of space between ":" and "C":

Reply:
The title is being retrieved directly from GitHub. I've just made an update to the title for https://github.com/dell/csm/issues/1030. The issue should be automatically resolved in the next pull request.